BSPConvert.Lib/Source/EntityConverter.cs

```diff
@@ -468,6 +471,65 @@ private void ConvertTriggerTargetsRecursive(Entity trigger, Entity entity)
 			}
 		}
 
+		private void ConvertTargetPushTrigger(Entity trigger, Entity targetPush)
+		{
+			var targets = GetTargetEntities(targetPush);
+			if (targets.Any())
+			{
+				trigger.ClassName = "trigger_jumppad";
+				trigger["launchtarget"] = targets.First().Name;
+				trigger["spawnflags"] = "1";
+			}
+			else
+			{
+				var launchVector = GetLaunchVector(targetPush);
+				var connection = new Entity.EntityConnection()
+				{
+					name = "OnStartTouch",
+					target = "player",
+					action = "SetLocalVelocity",
+					param = $"{launchVector.X} {launchVector.Y} {launchVector.Z}",
+					delay = 0,
+					fireOnce = -1
+				};
+				trigger.connections.Add(connection); ;
+			}
+		}
+
+		private static Vector3 GetLaunchVector(Entity targetPush)
+		{
+			var angles = "0 0 0";
+
+			if (!String.IsNullOrEmpty(targetPush["angles"]))
+				angles = targetPush["angles"];
+			else if (float.TryParse(targetPush["angle"], out var angle))
+				angles = $"0 {angle} 0";
+
+			var angleString = angles.Split(' ');
+			
+			var pitchDegrees = float.Parse(angleString[0]);
+			var yawDegrees = float.Parse(angleString[1]);
+			var rollDegrees = float.Parse(angleString[2]);
+
+			var yaw = Math.PI * yawDegrees / 180.0;
+			var pitch = Math.PI * -pitchDegrees / 180.0;
+			var roll = Math.PI * rollDegrees / 180.0;
+
+			var x = Math.Cos(yaw) * Math.Cos(pitch);
+			var y = Math.Sin(yaw) * Math.Cos(pitch);
+			var z = Math.Sin(pitch);
+
+			var unitVector = new Vector3((float)x, (float)y, (float)z);
```

Comment:
Lines 514 - 522 can be extracted into a reusable utility method. MathUtil.cs `ConvertAnglesToVector(float yaw, float pitch)`

Roll isn't used so you can ignore it.